packages/wp-dashboard/src/components/editorSettings/customFonts/confirmationDialog.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
@BrittanyIRL Tagging you here early on since this is a change for the Dashboard Settings. Let me know if you see anything that's out of place / should be improved.